src/main/java/com/synopsys/integration/detect/lifecycle/run/operation/OperationFactory.java

```diff
@@ -584,11 +586,15 @@ public BinaryScanOptions calculateBinaryScanOptions() {
     }
 
     public void publishBinaryFailure(String message) {
+        // TODO how should the arg be used? Ignoring it is hiding important error info; adding this for now:
```

Comment:
@stevebillings not sure what the question is here.
I don't see anything wrong with logging an error.

Are you suggesting else consume the published failure and log the error?